sdk/nodejs/cmd/run/index.ts

```diff
@@ -94,6 +94,7 @@ function usage(): void {
     console.error(`    where [flags] may include`);
     console.error(`        --organization=o    set the organization name to o`);
     console.error(`        --project=p         set the project name to p`);
+    console.error(`        --root_directory=p  set the project root directory (location of Pulumi.yaml) to p`);
```

Comment:
```suggestion
    console.error(`        --root-directory=p  set the project root directory (location of Pulumi.yaml) to p`);
```

kebab case for args, see query-mode and dry-run below